lab/lab1/part3-data-transformation.html

```diff
@@ -98,7 +109,13 @@
 
     ===================== */
 
+    var makeAMarker = (obj) => { //"obj" is the user input, in this case "obj" will be dataFiltered
+      return L.marker([obj[1], obj[0]]).bindPopup(obj[2]) //the "first" item of each sub-array isi the lat and the "zeroth" items for them are the long coordinates (IN JAVA AND GOOGLE MAPS, COORDINATES ARE MAPS AS LAT/LON FOR SOME REASON). The "second" item of each sub-array is the station's name.
+      }
 
+    for (var i=0; i <dataFiltered.length; i = i+1) {
+      makeAMarker(dataFiltered[i]).addTo(map) //Uses the makeAMarker function created above to add the points to the map
+    }
```

Comment:
You did not attempt to "clean" the dirty data file. That is the really challenging part about this lab (and the main reason for my request for cleanup)